org.qutebrowser.qutebrowser.json

```diff
@@ -0,0 +1,217 @@
+{
+    "app-id": "org.qutebrowser.qutebrowser",
+    "branch": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "qutebrowser-wrapper",
+    "finish-args": [
+        /* Nested sandboxes are currently not support by flatpak */
+        "--env=QTWEBENGINE_DISABLE_SANDBOX=1",
+        /* Allow direct rendering to use OpenGL and access to webcams */
+        "--device=all",
+        /* Allow to read all files from the home directory */
+        "--filesystem=home:ro",
```

Comment:
Qt5's standard file chooser should use the portal and this shouldn't be needed. I don't know if you are familiar with Qt but if this doesn't work that should be solved.

Reply:
@glatzor Have you looked at this?